test/e2e/fixtures.go

```diff
@@ -49,6 +49,12 @@ func skipIfAntreaIPAMTest(tb testing.TB) {
 	}
 }
 
+func skipIfNotFlowVisibilityTest(tb testing.TB) {
+	if !testOptions.flowVisibility {
+		tb.Skipf("Skipping test when not running flow visibility test: %s", tb.Name())
```

Comment:
nit: do we need to print test name here? the test framework would indicate which test is being skipped.

Reply:
removed. thx